modules/diskq/logqueue-disk-non-reliable.c

```diff
@@ -36,6 +36,10 @@ typedef struct
   LogQueue *queue;
 } DiskqMemusageLoaderState;
 
+
+/* helper function for _ack_backlog() */
+void _ack_backlog_one_message(LogQueueDiskNonReliable *self, gpointer data, LogPathOptions *path_options);
```

Comment:
Function declarations should be in header files, but from what I can see this function is not used anywhere. How about removing it?

Reply:
Hi, @OverOrion. Yeah I've implemented that to make the code cleaner (to have a function instead of macro revamp), but that would have required to change implementation at all the places, So instead to avoid code duplication I've revamped the macro as suggested my @MrAnno but forgot to remove the same declaration. Removed it 👍